launch/sim2play.launch.py

```diff
@@ -46,9 +52,18 @@ def generate_launch_description():
             "sim_flag": "-sim",
             "ref_flag": ref_flag,
             "direction_flag": "minus",
+            "use_internal_ref": use_internal_ref,
+            "use_sim_radio": use_sim_radio,
+            "config_yaml": config_yaml,
         }.items(),
     )
 
     blue = GroupAction([PushRosNamespace("blue"), soccer_blue])
 
-    return LaunchDescription([stdout_linebuf_envvar, yellow, blue])
+    return LaunchDescription(
+        [
+            stdout_linebuf_envvar,
+            yellow,
+            TimerAction(period=3.0, actions=[blue]),
```

Comment:
dumb fix for the sim2play non-deterministic startup problem. A better solution would be to create a custom action that waits for yellow to launch, then blue can launch. But since this works I don't feel like making it better lol.

Reply:
honestly this works great, good-enough hack imo